vpd-tool/meson.build

```diff
@@ -7,11 +7,12 @@ endif
 
 dependency_list = [CLI11_dep, sdbusplus]
 
-sources = ['src/vpd_tool_main.cpp']
+sources = ['src/vpd_tool_main.cpp',
+            'src/vpd_tool.cpp']
 
 vpd_tool_exe = executable('vpd-tool',
                           sources,
-                          include_directories : ['../include/'],
```

Comment:
Why do we want to remove the `../include` directory? Is it to remove `vpd-tool` dependency on `vpd-manager` ?

Reply:
yes, to remove any dependency on vpd-manager.
vpd-tool should have util API's, types defined in the vpd-tool build folder itself, it wont sharing anything from vpd-manager.

The needed utils APIs/constants/types will be replicated to vpd-tool. So vpd-tool and vpd-manager can be build independently .